BluePic-OpenWhisk/actions/Alchemy.swift

```diff
@@ -9,34 +9,35 @@ import RestKit
 import AlchemyVision
 
 func main(args:[String:Any]) -> [String:Any] {
-       
-    let alchemyKey: String? = args["alchemyKey"] as? String
-    let imageURL: String? = args["imageURL"] as? String
-
-
-    let alchemyVision = AlchemyVision(apiKey: alchemyKey!)
+    
+    let alchemyKey = args["alchemyKey"] as? String ?? ""
```

Comment:
So if this ends up being `""` is that ok? Do we just end up with a printed error?